docs/product_release.md

```diff
@@ -0,0 +1,44 @@
+# Product Release
+
+Tractus-X products do follow their own release process. Please check the leading product repositories for details.
+Independent of the technical details of product release processes, the released artifacts and minimum level of quality ensurance
+is the same across all of them.
+
+The release contents for individual products are are planned in the [open refinement and planning](./open-refinement-and-planning.md) sessions.
+These sessions define the scope for all dataspace components, but therefore paint a clear picture of enhancements necessary for every single product.
+The product teams themselfes are then responsible to define fine-grained task breakdowns as suiting their own needs.
+
+> __HINT:__
+> If product enhancements contribute to an overarching dataspace feature, please link the dataspace feature defined in `sig-release` in your product issue.
+> These links can be done in either the description or any comment on your issue. Use `eclipse-tractusx/sig-release#issue-number`
+
+Once planning activities for a certain iteration are completed, the implementation phase is starting and product teams can coordinate, where necessary
+through our [communication channels](https://eclipse-tractusx.github.io/docs/oss/getting-started#communication-channels).
+
+Integration tests are performed by maintaining an umbrella Chart, that deploys all relevant dataspace components to spin up an artificial dataspace based on Tractus-X components.
```

Comment:
is "Product Umbrella Chart" now the best term based on conversation below?
it would be separated from " Tractus-X Umbrella Chart", right?